### ISSUE TITLE
Hosted version

### ISSUE THREAD

**Issue by steren** [openfootprint/openfootprint#8]
Self hosting is great (and works like a charm thanks to Docker compose and the makefile), but what about offering openfootprint as a service?

Let me know if you need help running it on GCP.
The django frontend should be able to run on https://cloud.run, which can securely connect to a managed PostgreSQL Cloud SQL database (see [docs](https://cloud.google.com/run/docs/configuring/connect-cloudsql))

**Comment by sylvinus** [comment]
Yes we definitely want a hosted version. We've pushed on Heroku for now.

Happy to talk about GCP, at the very least to have it as an alternative option for users :)
